pages/linux/toolbox-rmi.md

```diff
@@ -0,0 +1,16 @@
+# toolbox rmi
+
+> Remove one or more `toolbox` images.
+> More information: <https://manned.org/toolbox-rmi.1>.
+
+- Remove a toolbox image:
+
+`toolbox rmi {{image_name}}`
```

Comment:
When you can create and remove containers, why can you only remove images? Where do they come from? And how do you add new ones?

Reply:
Image is like an iso of a VM you can always add and reinstall it in new disks and test drive it. Containers are dependent on images. So if you remove an image, dependent containers are removed too, we need to redownload the image and create new containers. 

rmi removes image alone with dependent containers whereas rm removes containers but the images are unaffected.